src/qs8-vmulc/rvv.c.in

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+#include <riscv_vector.h>
+#include <xnnpack/vbinary.h>
+
+$assert DATATYPE in ["QS8", "QU8"]
+$XINT8_T = {"QS8": "int8_t", "QU8": "uint8_t"}[DATATYPE]
+$VSIZE = {"f2": "mf2", 1: "m1", 2: "m2"}[LMUL]
+$VSIZE2 = {"f2": "m1", 1: "m2", 2: "m4"}[LMUL]
+$VSIZE4 = {"f2": "m2", 1: "m4", 2: "m8"}[LMUL]
```

Comment:
Can we align the usage of LMUL in existing `rvv.c.in`?
https://github.com/google/XNNPACK/blob/3b7f53d2cc59c9d58495da0662672aed1248d895/src/f32-vhswish/rvv.c.in#L36